codec-redis/src/main/java/io/netty/handler/codec/redis/RedisMessageType.java

```diff
@@ -69,7 +76,7 @@ public static RedisMessageType valueOf(byte value) {
         case '*':
             return ARRAY_HEADER;
         default:
-            throw new RedisCodecException("Unknown RedisMessageType: " + value);
+            return INLINE_COMMAND;
```

Comment:
We're changing a catch-all exception to an accept-all use case. I'm wondering if there's something more accurate we could do, for ex. only recognize `a-zA-Z` as the marker of an inline command and restore the `RedisCodecException` for all other cases?

Reply:
Redis commands can be [renamed](https://redis.io/topics/security#disabling-of-specific-commands) for security reasons. Substitute commands are not limited to `a-zA-Z` though:

redis.conf
```
rename-command PING もしもし # hello in Japanese
```

Invoke renamed command (works in `redis-cli`, too)
```
$ echo もしもし | nc localhost 6379
+PONG
```

I don't see any good reason to be more restrictive than Redis itself is.

Reply:
The intent was not to be more restrictive than Redis is. Since `rename-command` allows using any character in commands, there's unfortunately not much check we can do.